examples/model_compress/main_tf_quantizer.py

```diff
@@ -1,119 +0,0 @@
-from nni.compression.tensorflow import QAT_Quantizer
```

Comment:
why remove it ?

Reply:
This example has been moved to file QAT_torch_pruner.py